lib/lambda.js

```diff
@@ -411,7 +410,12 @@ function elasticApmAwsLambda (agent) {
     }
 
     return function wrappedLambda (event, context, callback) {
-      log.trace({ awsRequestId: context && context.awsRequestId }, 'lambda: fn start')
+      if (!(event && context && typeof callback === 'function')) {
```

Comment:
👍  for guarding against unknown datatypes here, per previous discussions on how things can go wrong without this.